src/System.Private.Uri/tests/FunctionalTests/UriTests.cs

```diff
@@ -3,11 +3,14 @@
 // See the LICENSE file in the project root for more information.
 
 using Xunit;
+using System.IO;
 
 namespace System.PrivateUri.Tests
 {
     public static class UriTests
     {
+        private static readonly bool s_IsWindowsSystem = Path.DirectorySeparatorChar == '\\';
```

Comment:
In other test suites, we've used RuntimeInformation for this instead of adding such statics, e.g. RuntimeInformation.IsOSPlatform(OSPlatform.Windows)